src/Interactive/HostTest/InteractiveHost.UnitTests.csproj

```diff
@@ -24,10 +24,13 @@
     <!--
       Not using ProjectReference since it does not execute the specified targets when building in VS.
       See https://github.com/microsoft/msbuild/issues/5434.
+
+      Note: do not set TargetFramework=net472 for InteractiveHost32. The project is not multi-targeted.
+      Setting the property would create a build configuration that's different from the one the solution uses
+      and thus would result in building the project twice.
     -->
     
     <CopyPublishedOutputProjectReference Include="..\HostProcess\InteractiveHost32.csproj">
-      <SetTargetFramework>TargetFramework=net472</SetTargetFramework>
```

Comment:
same for 64? ah, no, because it is multitargeted.

Reply:
Yup, that one is multi-targeted.